PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azuretools/ijidea/actions/SelectSubscriptionsAction.java

```diff
@@ -59,43 +60,10 @@ public SelectSubscriptionsAction() {
     @AzureOperation(value = "select one/more subscriptions to work on", type = AzureOperation.Type.ACTION)
     public boolean onActionPerformed(@NotNull AnActionEvent e, @Nullable Operation operation) {
         Project project = DataKeys.PROJECT.getData(e.getDataContext());
-        onShowSubscriptions(project);
+        selectSubscriptions(project).subscribe();
         return true;
```

Comment:
can we put subscribe() method into selectSubscriptions(...) ?
I think it‘s elusive for developers to call it.

Reply:
no, some other callings need to do something in `subscribe`, e.g.
![image](https://user-images.githubusercontent.com/69189193/103495832-08c2f000-4e77-11eb-87ab-3b88dd4150bd.png)

Reply:
how about adding anoter selectSubscriptions(myProject, subActions) ?